values.yaml

```diff
@@ -194,6 +194,14 @@ ldap:
   adminDn: "cn=admin,dc=georchestra,dc=org"
   rolesRdn: "ou=roles"
   orgsRdn: "ou=orgs"
+  # You can override in this secret some sensitive env. vars, like the ldap admin password
```

Comment:
Maybe add a note that it can NOT be changed just by changing the value of the secret !!!

Reply:
Yep, you're right. I'm reformulating.